packages/react/src/components/MDXComponents/components/_utils.scss

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright IBM Corp. 2022, 2025
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+$carbon-platform-prefix: 'carbon-platform-mdx-components';
+
+@function with-prefix($input) {
+  @return $carbon-platform-prefix + '--' + $input;
```

Comment:
Let's do a different prefix, since this isn't in the `carbon-platform` repo anymore. How about `$carbon-mdx`?